__tests__/app.test.js

```diff
@@ -11,4 +13,95 @@ describe('secret-scratch routes', () => {
   afterAll(() => {
     pool.end();
   });
+
+  it('signs up a user via POST', async () => {
+    const res = await request(app)
+      .post('/api/v1/users/register')
+      .send({ email: 'chase@gmail.com', password: 'password' });
+
+    expect(res.body).toEqual({
+      id: expect.any(String),
+      email: 'chase@gmail.com',
+    });
+  });
+
+  it('signs in an existing user', async () => {
+    const user = await UserService.create({
+      email: 'chase@gmail.com',
+      password: 'password',
+    });
+
+    const res = await request(app)
+      .post('/api/v1/users/session')
+      .send({ email: 'chase@gmail.com', password: 'password' });
+    expect(res.body).toEqual({
+      message: 'Sign in Successful',
+      user,
+    });
+  });
+
+  it('allows the `user` user to view a list of posts', async () => {
+    const agent = request.agent(app);
+
+    // await UserService.create({
+    //   email: 'admin',
+    //   password: 'admin',
+    // });
+
+    await UserService.create({
+      email: 'chase@gmail.com',
+      password: 'password',
+    });
+
+    let res = await agent.get('/api/v1/posts');
+    expect(res.status).toEqual(401);
+
+    await agent
+      .post('/api/v1/users/session')
+      .send({ email: 'chase@gmail.com', password: 'password' });
+    res = await agent.get('/api/v1/posts');
+    expect(res.status).toEqual(200);
```

Comment:
You should create a couple of posts, and in addition to the 200 assertion, ensure that your two created posts are sent in the response.